packages/bpk-component-map/src/BpkPriceMarker.js

```diff
@@ -0,0 +1,131 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2016-2020 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/* @flow strict */
+
+import React from 'react';
+import PropTypes from 'prop-types';
+import { cssModules } from 'bpk-react-utils';
+import ArrowDownIcon from 'bpk-component-icon/lg/arrow-down';
+import BpkText, {
+  WEIGHT_STYLES,
+  TEXT_STYLES,
+} from 'bpk-component-text/src/BpkText';
+
+import { LatLongPropType, type LatLong } from './common-types';
+import BpkBasicMapMarker from './BpkBasicMapMarker';
+import STYLES from './BpkPriceMarker.scss';
+
+const getClassName = cssModules(STYLES);
+
+export const MARKER_STATUSES = {
```

Comment:
Can we name this `PRICE_MARKER_STATUSES`? It'll make it clearer to consumers that it applies to the price marker component, otherwise they might think it's something to be used with `BpkMapMarker`.

Sidenote: You've already named it `PRICE_MARKER_STATUSES` a couple of times in the readme, probably copypasta'd from the RN readme?

Reply:
Sure thing sounds good!